components/Note/src/note.css

```diff
@@ -0,0 +1,45 @@
+.denhaag-note {
+  --denhaag-note-icon-color: #4b4b4b;
```

Comment:
To tokens.

Reply:
I did'nt touch the note.css, besides this is not the posttype label component.